examples/bokeh.html

```diff
@@ -31,7 +31,9 @@
 
             <py-config>
               packages = [
-                "bokeh"
+                "pandas",
```

Comment:
With the new version of bokeh, do we have to install dependencies separate? Or is it because of the micropip changes?

Reply:
Fixed this too by using the link suggested by @philippjfr